### DISCUSSION TITLE
Does video resolution "source" always mean best quality?

### DISCUSSION THREAD

**Discussion by TheMissingPort** [DIGITALCRIMINALS/OnlyFans#771] [General]
For example I'm downloading videos from Fansly that have 4k, but the file size doesn't seem to add up to 4k resolution size. its a three minute vid that only adds up to 300mbs when it should be more. Any ideas? Thanks

**Comment by DIGITALCRIMINAL** [comment]
Yes, that's the intended behaviour, to get the quality as close to the source as possible.

I'll have to fix fansly if that's not the case.
